docs/dev/release.md

```diff
@@ -0,0 +1,34 @@
+Making releases
+===============
+
+The following examples will assume that we are releasing `v2.0.0`.  Use the correct release version when making the actual release.
+
+Release Hummingbird
+-------------------
+
+    1.  Make a `v2.0.0` tag of hummingbird master:
+        1.  `git checkout master`
+        2.  `git pull upstrem master`
+        3.  `git push origin v2.0.0`
+        4.  `git push --tags`
+    2.  CI will build binaries and create the github release.
+    3.  Edit title and create release notes for the new release at <https://github.com/troubling/hummingbird/releases/tag/v2.0.0>.
+
+Release Hansible
+----------------
+
+    1.  Update `hummingbird_version` to `v2.0.0` in `roles/humminbird-common/defaults/main.yml` and `group_vars/hummingbird/hummingbird` and push to master.
+    2.  Draft a new release at <https://github.com/troubling/hansible/releases/new>.
+    3.  Set Tag version and release title to `v2.0.0`.
+    4.  Fill in release notes and publish the release.
+    5.  Change `hummingbird_version` back to `dev` in `roles/humminbird-common/defaults/main.yml` and push to master.
+
+Release RPC-Hummingbird
+-----------------------
+
+    1.  Update hansible version to `v2.0.0` in `ansible-role-requirements.yml` and push to master.
+    2.  Make a pull request to https://github.com/rcbops/releases:
+        1.  add new version and git sha to `components/rpc-hummingbird.yml`.
```

Comment:
Whether you wish to use is up to you, but just documenting this so you know. We have a tool at https://github.com/rcbops/rpc_component (requires python3) which you can use to modify the component releases file. At first hand it seems overkill, but it does some basic validation when modifying the file, and ensures releases are added in such a way to avoid merge conflicts if multiple things are releasing at the same time (a potential problem for rpc-openstack that has multiple different series that release simultaneously). It also handles committing the change so commit messages, etc. will be standardised when you do these releases. To add the release, cd into the releases checkout and:

```cd releases; component --releases-dir . release --component-name rpc-hummingbird add --version <version> --series-name master --sha <sha>```